### DISCUSSION TITLE
`/var/folders/...` present in empty fake fs

### DISCUSSION THREAD

**Discussion by tfeldmann** [jmcgeheeiv/pyfakefs#884] [Q&A]
**Describe the bug**
A directory `var/folders/` is present on empty fake filesystems, containing several files and subfolders.

**How To Reproduce**
```python
import os

def test_simple(fs):
    for x in os.scandir("."):
        assert "var" not in x.name
```

**Your environment**
```
macOS-13.5.1-arm64-arm-64bit
Python 3.11.5 (main, Aug 24 2023, 15:09:45) [Clang 14.0.3 (clang-1403.0.22.14.1)]
pyfakefs 5.2.4
pytest 6.2.5
```

**Comment by mrbean-bremen** [comment]
This is due to the [presence of the temp directory](https://pytest-pyfakefs.readthedocs.io/en/latest/troubleshooting.html#os-temporary-directories) - the `tempfile` module relies on that existing.

**Comment by tfeldmann** [comment]
Thank you, somehow I missed this. I moved the tests into a subfolder. Feel free to close 👍

**Comment by mrbean-bremen** [comment]
Moving to discussion, as this may be missed by others...
